eng/Version.Details.xml

```diff
@@ -25,6 +25,21 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>d099f075e45d2aa6007a22b71b45a08758559f80</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.Extensions.Logging" Version="7.0.0">
+      <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>d099f075e45d2aa6007a22b71b45a08758559f80</Sha>
+      <SourceBuild RepoName="runtime" ManagedOnly="true" />
+    </Dependency>
+    <Dependency Name="Microsoft.Extensions.Logging.Abstractions" Version="7.0.0">
+      <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>d099f075e45d2aa6007a22b71b45a08758559f80</Sha>
+      <SourceBuild RepoName="runtime" ManagedOnly="true" />
```

Comment:
There can be only one `Dependency` which has a `SourceBuild` child element for a given repo name (e.g. "runtime"). It's purely a marker that indicates that `runtime` will be used for source build. It doesn't matter which dependency it's attached to, but it can only be attached to one. This means you can delete the `SourceBuild` element here and on the dependency below.